index.js

```diff
@@ -224,8 +223,8 @@ function fetch(...args:any):Promise {
 
   // # 241 normalize null or undefined headers, in case nil or null string
   // pass to native context
-  headers = _.reduce(headers, (result, value, key) => {
-    result[key] = value || ''
+  headers = headers && Object.keys(headers).reduce((result, key) => {
+    result[key] = headers[key] || ''
     return result
   }, {});
```

Comment:
1. no need to import whole lodash:
```javascript
import { reduce } from 'lodash'
```
2. lodash functions will handle it even better:
```javascript
import { mapValues, toString } from 'lodash'
// ...
headers = headers && mapValues(headers, toString)
```